src/api/friend.js

```diff
@@ -0,0 +1,54 @@
+import axios from "../api/axiosInstance";
+
+const useMock = true;
+
+const numbers = 16;
+
+const mockUser = {
+  profileUrl: "https://lh3.googleusercontent.com/a/AATXAJyVWL-UkAOLeLY75qUbVwQWvUt1RDWk60KkzICi=s96-c0",
+  name: "user mock",
+  score: 9,
```

Comment:
서버쪽에서 scores 스키마 변경 PR을 작성해주신 것과 동일하게 객체로 수정되면 좋을 것 같습니다.
```suggestion
  scores: {
    activity: 0,
    meal: 4.25,
    sleep: 0,
    바코9기사진첩: 1.8333333333333333,
    바코10기사진첩: 3.56,
    grid: 3,
  },
```

Reply:
아직 그부분은 합의된 바가 없습니다!
일단 객체로 변경해서 평균점 계산하는 로직 추가하였습니다.

Reply:
그렇군요. 친구 전체 페이지에서는 개별 항목 점수는 필요가 없으니 서버에서 계산해서 보내줘도 좋을 것 같네요. 반영해주셔서 감사합니다.